### DISCUSSION TITLE
New Issued Document (Invoice) mandatory parameters

### DISCUSSION THREAD

**Discussion by programmerxx89** [fattureincloud/api#359] [Q&A] [answered]
Hi,
I'm creating an Issued Document (in particular an invoice) using Zapier.
The list of parameters that I can provide to the FattureInCloud API is huge, and I think that I'm having problems running that zap because not all mandatory fields are filled.
Which are the mandatory fields of an Issued Document to be filled when I want to create an invoice?

Thanks

**Comment by emajo** [comment, marked as answer]
Hi @programmerxx89, the only fields that are mandatory when creating an invoice are the `type` ad the `entity.name` fields, but a lot of other fields can be mandatory based on the fields you are setting in the request (for example if you set items_list you must also set payments_list).

In the error you recive usually is specified which field is missing in your request, if you are not receiving any error I suggest to write to our customer support so we can check with more information.

Thanks,
Emanuele.

**Comment by programmerxx89** [comment]
can you please tell me how I can write to your customer support?

**Reply by emajo** [reply]
You can find the chat in every Fatture in Cloud page or at the bottom of [this page](https://secure.fattureincloud.it/help/).
